### ISSUE TITLE
Rule "comma-dangle" does not apply to ES2015 import statements

### ISSUE THREAD

**Issue by joshforisha** [eslint/eslint#3794]
import { apple, orange, } from 'fruits';

does not trigger rule `comma-dangle": [ 2, "never" ]`.

I'm using the current stable version, 1.4.1.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by btmills** [comment]
Verified in the demo that all cases below are caught except `import`.

```js
import { apple, orange, } from "fruits";
var c = [ 1, 2, ];
var d = { a: 1, b: 2, };
var [e, f, ] = c;
var { a, b, } = d;
```

**Comment by michaelficarra** [comment]
The same probably applies for the various export forms that include export specifiers.

**Comment by mysticatea** [comment]
I will work on this.
